src/cmd-build

```diff
@@ -462,18 +469,35 @@ else
                  "--label=io.openshift.build.versions=machine-os=${buildid}"
                 )
     fi
+    
+    last_build_manifest=()
+    if rpm-ostree compose container-encapsulate --help |grep -q -e "--previous-build-manifest"; then
```

Comment:
I think actually nowadays we *can* rely on having a new enough rpm-ostree here, so we could probably now drop this check.